src/ui/studio/save-creation/index.js

```diff
@@ -419,46 +440,53 @@ const UploadForm = ({ uploadState, handleUpload }) => {
       </Trans>
     );
 
-  return (
-    <Fragment>
-      <NotConnectedWarning />
-
-      <form>
-        { titleField !== FORM_FIELD_HIDDEN && <Input
-          name="title"
-          label={t('save-creation-label-title')}
-          required={titleField === FORM_FIELD_REQUIRED}
-          onChange={handleInputChange}
-          autoComplete="off"
-          defaultValue={title}
-          {...{ errors, register }}
-        /> }
-
-        { presenterField !== FORM_FIELD_HIDDEN && <Input
-          name="presenter"
-          label={t('save-creation-label-presenter')}
-          required={presenterField === FORM_FIELD_REQUIRED}
-          onChange={handleInputChange}
-          autoComplete="off"
-          defaultValue={presenterValue}
-          {...{ errors, register }}
-        /> }
-
-        <Button
-          disabled={recordings.length === 0}
-          onClick={handleSubmit(handleUpload)}
-        >
-          <FontAwesomeIcon icon={faUpload} />
-          { buttonLabel }
-        </Button>
-      </form>
+  const handlers = {
+    UPLOAD: (keyEvent) => { if(keyEvent) {document.getElementById("upload").click()}},
+  }
 
-      <Box sx={{ mt: 2 }}>
-        { uploadState.state === STATE_ERROR && (
-          <Notification isDanger>{uploadState.error}</Notification>
-        )}
-      </Box>
-    </Fragment>
+  return (
+    <GlobalHotKeys keyMap={keyMap} handlers={handlers} allowChanges={true}>
```

Comment:
Done. 
Removed `allowChanges={true}` from component and added `ignoreKeymapAndHandlerChangesByDefault: false` to keymap config. Since there are now multiple keyMaps with different names, I only pass the needed keyMap.